plugins/polio/api/vaccines/vaccine_authorization.py

```diff
@@ -247,6 +253,24 @@ def get_most_recent_authorizations(self, request):
 
                 response.append(vacc_auth)
 
+        if auth_status:
+            response = [entry for entry in response if entry["status"] in auth_status.split(",")]
+
+        if block_country:
+            block_country = block_country.split(",")
+            block_country = Group.objects.filter(pk__in=block_country)
+            org_units = [
```

Comment:
There are three imbricated for loops on one line. could it be possible to make separated for loops and each in its variable to make the code more readable

Reply:
To me it's one of the best advantage of the list comprehension in Python is that you can shrink simple for loop.

Reply:
Indeed after reading it I kinda wrote it too quickly and can simplify it